ci-operator/step-registry/openstack/conf/resourcenames/openstack-conf-resourcenames-commands.sh

```diff
@@ -168,3 +168,20 @@ CLUSTER_NAME: $(cat "${SHARED_DIR}/CLUSTER_NAME")
 ZONES: $ZONES
 BASTION_FLAVOR: $BASTION_FLAVOR
 EOF
+
```

Comment:
openstack-reviewers: I know this place isn't ideal but I haven't found anywhere else to put this, unless we want to create a new step for this, which means including it to the chains, which means duplicating the chains to have it in the NFV workflows.

I don't like duplicating the chains because they're often out-of-sync from the reference and main chain; so I would rather have this known & documented workaround here, to avoid hundreds of json & yaml in this repo.

Reply:
Another option (brought by @wking) is to use a variable instead of a file, and add the variable in all the jobs, but that requires patching 50 NFV jobs, not ideal either.

I know this solution is not pretty but I hope this comment makes it less magical and hopefully temporary.